### PR TITLE
(rancher.ls) into fleet

### PR DESCRIPTION
**Pull request by cbarria**
Moving rancher.ls into fleet! 👍🏻

### CODE REVIEW

File: fleet/lib/fleet-conf/overlays/ls/gitrepo-rancher.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: fleet.cattle.io/v1alpha1
+kind: GitRepo
+metadata:
+  name: rancher
+  namespace: fleet-local
+spec:
+  repo: https://github.com/lsst-it/k8s-cookbook
+  branch: master
```
Comment (jhoblitt) [inline fleet/lib/fleet-conf/overlays/ls/gitrepo-rancher.yaml:9]: Clusters in `ls` should be using the `ls_production` branch.

File: fleet/lib/metallb-conf/overlays/rancher.ls/ipaddresspool-ingress.yaml
Comment (jhoblitt) [inline fleet/lib/metallb-conf/overlays/rancher.ls/ipaddresspool-ingress.yaml:1]: A change to `metallb-conf/fleet.yaml` is required for this have any effect.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: fleet/lib/fleet-conf/overlays/ls/gitrepo-rancher.yaml (hunk 1 above)
Comment (cbarria) [reply fleet/lib/fleet-conf/overlays/ls/gitrepo-rancher.yaml]: changed to ls_production 👍🏻  my bad

File: fleet/lib/metallb-conf/overlays/rancher.ls/ipaddresspool-ingress.yaml
Comment (cbarria) [reply fleet/lib/metallb-conf/overlays/rancher.ls/ipaddresspool-ingress.yaml:1]: added rancher.tu and ls 👍🏻

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Pull request merged**